src/fbrs/controller/MarketsController.java

```diff
@@ -30,6 +56,43 @@ public void printAllMarkets() throws IOException {
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
+        List<Market> markets = model.getAllMarkets();
+
+        // Create a pane and set its properties
+        gridPane.setAlignment(Pos.CENTER);
+        gridPane.setPadding(new Insets(8, 0, 0, 0));
+        gridPane.setVgap(64);
+        gridPane.setHgap(64);
+
+        // Place nodes in the pane
+        for (int i = 0; i < markets.size(); i++) {
+            if (i < markets.size() / 2)
+                gridPane.add(createMarketButton(markets.get(i)), 0, i);
+            else
+                gridPane.add(createMarketButton(markets.get(i)), 1, i - markets.size() / 2);
+        }
```

Comment:
gridPane.add(createMarketButton(markets.get(i)), 0, i);
can be replaced by
`gridPane.add(createMarketButton(markets.get(i)), i%2, i/2);`
or you can define the number of columns and change `i%2` and `i/2` to `i%columnCount` and` i/columnCount`